deployment/dev.env

```diff
@@ -0,0 +1,37 @@
+# This file holds an example of the environment variables that should be
+# present in order to successfully deploy this project.
+#
+# This file can be used by running the following sequence of bash commands:
+#
+# set -o allexport
+# source dev.env
+# set +o allexport
+#
+#
+# These commands will temporarily put bash in a mode that automatically
+# exports all variable definitions so that they are inherited by child
+# processes, then source this file and finally restore the default (don't
+# automatically export variables) behaviour
+#
+#
+# ######## WARNING ##########################################################
+#
+# THE FOLLOWING VARIABLES ARE NOT SUITABLE FOR USE IN PRODUCTION NOR ON PUBLIC
+# TESTING MACHINES.
+#
+# They may be appropriate for a standalone (offline) development environment.
+#
+#############################################################################
+
+DJANGO_SETTINGS_MODULE='statuto-territorio-rt.settings'
+DJANGO_ALLOWED_HOSTS='127.0.0.1 localhost' 
+DJANGO_DEBUG='True'
+DJANGO_SECRET_KEY='&z+bl66a=!^9#njhsxwcq8n#)hc2n_isw%x0w5ir%3)gk7w8!j'
```

Comment:
Be careful to never expose real keys / usernames / password on git commits

Reply:
these are fake keys @afabiani that's why I asked to adopt env vars